apps/mainsite/settings.py

```diff
@@ -254,44 +254,34 @@
 ##
 
 LOGGING = {
-    "version": 1,
-    "disable_existing_loggers": False,
-    "handlers": {
-        "mail_admins": {
-            "level": "ERROR",
-            "filters": [],
-            "class": "django.utils.log.AdminEmailHandler",
-        },
```

Comment:
I suppose if we take this out (which is a good idea), we should probably look into getting notified by grafana right? Sounds like its own story though.
What do you think?